eldritch/encounters.py

```diff
@@ -262,6 +262,111 @@ def Square7(char):
+  check = events.Check(char, "luck", -1)
+  success = events.Nothing()
+  fail = events.Arrested(char)
+  passfail = events.PassFail(char, check, success, fail)
+  return events.Sequence([items, passfail], char)
+def Docks3(char):
+  check = events.Check(char, "fight", 0)
+  dollars = events.Gain(char, {"dollars": 3}) # TODO: This is really dollars * number of successes
+  move = events.ForceMovement(char, "Merchant")
+  stamina = events.Loss(char, {"stamina": 1})
+  cond= events.Conditional(char, check, "successes", {0: events.Sequence([stamina, move], char), 1: dollars})
+  return events.Sequence([check, cond], char)
+def Docks4(char):
+  check = events.Check(char, "will", -1)
+  item = events.Draw(char, "unique", 1)
+  success = events.Sequence([events.Loss(char, {"sanity": 1}), item], char)
+  fail = events.Sequence([events.Loss(char, {"sanity": 2}), item], char)
+  return events.PassFail(char, check, success, fail)
+def Docks5(char):
+  check = events.Check(char, "speed", -1)
+  loss = events.Loss(char, {"sanity": 1})
+  return events.PassFail(char, check, events.Nothing(), loss)
+def Docks6(char):
+  check = events.Check(char, "will", 1)
+  lost = events.Nothing() # TODO: Oh No, what time is it?  Where am I?  I'm lost in Time AND Space!
+  return events.PassFail(char, check, events.Nothing(), lost)
+def Docks7(char):
+  check = events.Check(char, "luck", -1)
+  draw = events.Draw(char, "common", 1)
+  struggle = events.Loss(char, {"stamina": 3, "sanity": 1})
+  return events.PassFail(char, check, draw, struggle)
+
+def Unnamable1(char):
+  loss = events.Loss(char, {"sanity": 2})
+  # TODO: add prerequisite of ally being in deck
+  ally = events.DrawSpecific(char, "allies", "Brave Guy")
+  listen = events.Sequence([loss, ally], char)
+  return events.BinaryChoice(char, "Listen to the tale?", "Yes", "No", listen, events.Nothing())
+def Unnamable2(char):
+  check = events.Check(char, "lore", -1)
+  spell = events.Draw(char, "spells", 1)
+  clues = events.Gain(char, {"clues": 2})
+  delayed = events.LoseTurn(char)
+  read = events.PassFail(char, check, spell, events.Sequence([clues, delayed], char))
+  return events.BinaryChoice(char, "Read the manuscript?", "Yes", "No", read, events.Nothing())
+def Unnamable3(char):
+  # TODO: WHAT IS THAT THING??? IT LOOKS LIKE A GATE!
+  return events.Nothing()
+def Unnamable4(char):
+  check = events.Check(char, "speed", -1)
+  move = events.ForceMovement(char, "Merchant")
+  loss = events.Loss(char, {"stamina": 2})
+  return events.PassFail(char, check, move, loss)
+def Unnamable5(char):
+  check = events.Check(char, "luck", -1)
+  unique = events.Draw(char, "unique", 1)
+  loss = events.Loss(char, {"sanity": 1, "stamina": 2})
+  return events.PassFail(char, check, unique, loss)
+def Unnamable6(char):
+  check = events.Check(char, "speed", -1)
+  lost = events.Nothing() #TODO: I'm stuck inside a clock and I'm somewhere around Mars?
+  return events.PassFail(char, check, events.Nothing, lost)
```

Comment:
This one should be events.Nothing() instead of events.Nothing

Reply:
oops, that's what's happened when you don't have a unit test (and an uncompiled language)